analytic_engine/src/instance/mod.rs

```diff
@@ -152,6 +152,8 @@ pub struct Instance {
     pub(crate) replay_batch_size: usize,
     /// Write sst max buffer size
     pub(crate) write_sst_max_buffer_size: usize,
+    /// Min rows required when build sst filter
+    pub(crate) min_value_num_when_build_filter: usize,
```

Comment:
After some thoughts, I guess what we should avoid is to build a bloom filter for too many keys, which costs too much storage but provides high false positive rate.